api/db-cdr/changelog-schema/db.changelog-25.xml

```diff
@@ -17,10 +17,16 @@
       <column name="question_concept_id" type="INTEGER">
         <constraints nullable="false"/>
       </column>
-      <column name="path" type="INTEGER">
+      <column name="survey_order_number" type="INTEGER">
         <constraints nullable="false"/>
       </column>
-      <column name="is_main" type="BOOLEAN">
+      <column name="question_order_number" type="INTEGER">
+        <constraints nullable="false"/>
+      </column>
+      <column name="path" type="VARCHAR(1000)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="sub" type="INTEGER">
```

Comment:
Liquibase should be used to incrementally change tables using different change sets. We should not be changing existing files every time the schema changes. These columns should be added in another changelog file.  Example: db.changelog-26.xml
```
<changeSet author="srushtigangireddy" id="changelog-26">
   <dropColumn tableName="survey_question_map" columnName="is_main"/>
    <addColumn tableName="survey_question_map">
      <column name="survey_order_number" type="INTEGER"/>
      <column name="question_order_number" type="INTEGER"/>
      <column name="sub" type="INTEGER"/>
    </addColumn>
    <modifyDataType tableName="survey_question_map" columnName="path" newDataType="VARCHAR(1000)"/>
</changeSet>
```

Reply:
Addressed thanks.